controllers/sync/policy_spec_sync.go

```diff
@@ -74,7 +77,7 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 				},
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      request.Name,
-					Namespace: request.Namespace,
+					Namespace: r.TargetNamespace,
```

Comment:
Since this replacement is happening, shouldn't this also log using `r.TargetNamespace` rather than `request.Namespace`?